spinalcordtoolbox/scripts/sct_run_batch.py

```diff
@@ -55,21 +55,24 @@ def get_parser():
                         'file are the same as the command line arguments, except all dashes (-) are replaced with '
                         'underscores (_). Using command line flags can be used to override arguments provided in '
                         'the configuration file, but this is discouraged. Please note that while quotes are optional '
-                        'for strings in YAML omitting them may cause parse errors.\n' + dedent(
+                        'for strings in YAML omitting them may cause parse errors.\n'
+                        'Note that for the \'exclude_list\' (or \'include_list\') argument you can exclude/include '
+                        'entire subjects or individual sessions; see examples below.\n' + dedent(
```

Comment:
I'm not sure if it's useful, but technically you can also exclude/include entire sessions (across all subjects).

Reply:
Good point! I am aware of this. Maybe we can mention this within this "Note" sentence. @sandrinebedard, IIRC you are filtering your Standford data based on sessions (brain vs spinal cord). Could you please add an example syntax here?

Reply:
just added a suggestion for this! I still wrote `ses-03`, just to be consistent, but the concept is the same!